docs/workflow/sdk-web-form-reference.mdx

```diff
@@ -13,16 +13,12 @@ A custom form element can be styled using CSS in a few different ways:
 Inline CSS is the simplest way to style your elements, but doesn't have the same flexibility as a CSS style sheet.
```

Comment:
Similar question - wondering if we should have some kind of archived version of the old details, or if we're happy to just remove it entirely.